src/clj/qbits/spandex.clj

```diff
@@ -202,30 +203,66 @@
                (response-headers response)
                (.getHost response))))
 
+(defn response-ex->response
```

Comment:
I merged a modified/simplified version of this PR earlier :). including this function